### PR TITLE
0.3

### PR DESCRIPTION
**Pull request by lveyde**

### CODE REVIEW

**Review by djzager — APPROVED** [review]
Looks right to me.

File: deploy/kubevirt-operator.operatorgroup.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (djzager) [inline deploy/kubevirt-operator.operatorgroup.yaml:1]: I had been trying to figure out why this was an addition, when it is already on master, now I see that you moved it under `deploy/` which makes sense. Can you update this to be a move from `kubevirt-operator.operatorgroup.yaml --> deploy/kubevirt-operator.operatorgroup.yaml` ?

**Review by djzager — CHANGES_REQUESTED** [review]
One change request.

File: deploy/kubevirt-operator.operatorgroup.yaml (hunk 1 above)
Comment (lveyde) [reply deploy/kubevirt-operator.operatorgroup.yaml:1]: It's already kind'a is, look at the:
https://github.com/djzager/kubevirt-olm/pull/7/commits/364868e0f6fefc085ee5380abc689c789123f767

**Pull request merged**